.travis.yml

```diff
@@ -1,28 +1,71 @@
-dist: trusty
-sudo: required
+packagecloud: &packagecloud
+  provider: packagecloud
+  username: cs50
+  repository: libcs50
+  token: $PACKAGECLOUD_TOKEN
+  local-dir: build/deb
+  package_glob: *.deb
+  skip_cleanup: true
```

Comment:
No need?

Reply:
The `build` directory is not part of the repo. It's a build artifact. Unless this is set to true, I think Travis CI is gonna clean up before deployment, removing the deb.